code/modules/clothing/suits/armor.dm

```diff
@@ -441,16 +455,17 @@
 
 /obj/item/clothing/suit/armor/f13/legrecruit/vet
 	name = "legion veteran armor"
+	desc = "A more refined version of the recruit armor, this one features forged metal plates in key areas along with the removal of unnessceary weight from unimportant areas, allowing better freedom of movement in combat."
 	item_state = "legveteran"
 	armor = list("melee" = 25, "bullet" = 25, "laser" = 16, "energy" = 16, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 
 /obj/item/clothing/suit/armor/f13/legvexil
 	name = "legion vexillarius armor"
-	desc = "The armor appears to be based off of a suit of Legion veteran armor, with the addition of circular metal plates attached to the torso, as well as a banner displaying the flag of the Legion worn on the back."
+	desc = "The armor appears to be based off of a suit of legion veteran armor, with the addition of circular metal plates attached to the torso, as well as a banner displaying the flag of the Legion worn on the back."
```

Comment:
Legion is a proper name. Keep it capitalized. "A legion" and "The Legion" [Caesar's] are two different things.